README.md

```diff
@@ -79,6 +79,22 @@ Now you can use it:
 </button>
 ```
 
+#### Handle current status as boolean
```

Comment:
Just add it to the table of outputs

Reply:
Sorry, I can't find it: I found only Inputs table...

Reply:
Add one 😀

Reply:
Created new Outputs table. Let me know.